packages/app-store/discord/config.json

```diff
@@ -16,7 +16,7 @@
       "label": "{TITLE}",
       "linkType": "static",
       "organizerInputPlaceholder": "https://discord.gg/420gg69",
-      "urlRegExp": "^http(s)?:\\/\\/(www\\.)?discord.gg\\/[a-zA-Z0-9]*"
+      "urlRegExp": "^http(s)?:\\/\\/(www\\.)?(discord.gg|discord.com)\\/[a-zA-Z0-9]*"
```

Comment:
@hkirat Solution works but it could have been improved by doing the following `(discord.gg|discord.com)` to `discord.(gg|com)`

Reply:
Some more detailed cases I have managed to cover under: https://github.com/calcom/cal.com/pull/8612